apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -872,4 +878,210 @@ describe('RubricContainer', () => {
+    const sendEventSpy = sinon.spy(analyticsReporter, 'sendEvent');
+    stubFetchEvalStatusForUser(readyJson);
+    stubFetchEvalStatusForAll(readyJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: null});
+
+    render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
+
+    expect(sendEventSpy).to.have.been.calledWith(
+      EVENTS.TA_RUBRIC_TOUR_STARTED,
+      {}
+    );
+
+    sendEventSpy.restore();
+  });
+
+  it('sends event when user clicks next and back buttons', async function () {
+    const sendEventSpy = sinon.spy(analyticsReporter, 'sendEvent');
+    stubFetchEvalStatusForUser(readyJson);
+    stubFetchEvalStatusForAll(readyJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: null});
+
+    const {getByText} = render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
+
+    const nextButton = getByText('Next Tip');
+
+    fireEvent.click(nextButton);
+
+    await wait();
+
+    expect(sendEventSpy).to.have.been.calledWith(EVENTS.TA_RUBRIC_TOUR_NEXT, {
+      step: 0,
+      nextStep: 1,
+    });
+
+    const backButton = getByText('Back');
+
+    fireEvent.click(backButton);
+
+    await wait();
+
+    expect(sendEventSpy).to.have.been.calledWith(EVENTS.TA_RUBRIC_TOUR_BACK, {
+      step: 1,
+      nextStep: 0,
+    });
+
+    sendEventSpy.restore();
+  });
+
+  it('sends event when user exits the tour', async function () {
+    const sendEventSpy = sinon.spy(analyticsReporter, 'sendEvent');
+    stubFetchEvalStatusForUser(readyJson);
+    stubFetchEvalStatusForAll(readyJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
+    stubFetchTeacherEvaluations(noEvals);
+    stubFetchTourStatus({seen: null});
+
+    const {getByText} = render(
+      <Provider store={store}>
+        <RubricContainer
+          rubric={defaultRubric}
+          studentLevelInfo={defaultStudentInfo}
+          teacherHasEnabledAi={true}
+          currentLevelName={'test_level'}
+          reportingData={{}}
+          open
+        />
+      </Provider>
+    );
+
+    await wait();
+
+    // Had to access this button indirectly because RTL can't find it by text
+    const skipButton = getByText(
+      'Getting Started with Your AI Teaching Assistant'
+    ).closest('div').childNodes[1];
```

Comment:
This is a fine workaround, but I got curious and took a crack at debugging. I found the following makes it work the RTL way:
```suggestion
    const skipButton = await screen.findByRole('button', {name: '×'});
```
there are several tricky bits here:
1. the contents of the button is not an `x` as in `xyz` but a special character `×`
2. findByRole waits for certain elements to appear via promise, while getByRole checks immediately without waiting
3. because findByRole returns a promise, you have to `await` it